src/rendering/renderers/gl/renderTarget/GlRenderTargetAdaptor.ts

```diff
@@ -165,7 +165,7 @@ export class GlRenderTargetAdaptor implements RenderTargetAdaptor<GlRenderTarget
         // we are rendering to the main canvas..
         const colorTexture = renderTarget.colorTexture;
 
-        if (CanvasSource.test(colorTexture.resource))
+        if (colorTexture instanceof CanvasSource)
```

Comment:
What was the purpose of this change? I know we have talked about it before but I can't remember it properly

Reply:
good questions - this change allows all canvas sources to have framebuffers attached to them (not just the ones we use for rendering to the screen) . In this instance it means we cna directly modify the text canvas texture, without needing to duplicate or make a new one